src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
@pieterlukasse actually I don't think what I said is the same - the current solution is, if the `rowData` is an array, loop through it and call `download` on each of its elements, then combine them. What I'm suggesting is, call `download` on `rowData`, always, and deal with whatever it returns.

I think a good way to handle the multiple desired download formats could be: put a prop on `LazyMobXTable` called `download` which takes the result of calling each of the column `download` functions on all the data, and decides how to combine them.

So each row gives an array of either `string` or `string[]`, i.e. `(string|string[])[]`, thus the input to `LazyMobXTable.props.download` would be of type `(string|string[])[][]`